### DISCUSSION TITLE
Mathematical Table Filtering

### DISCUSSION THREAD

**Discussion by TravisCouture** [h2oai/wave#632] [Ideas]
#### Describe the solution you'd like

The ability to filter a table based on mathematical expressions on columns. E.g. (C1 > n)

#### Describe alternatives you've considered

Can be done using a form and manipulating the underlying dataframe based on the form input but this feels clunky. I think this should be a native feature of Wave tables.

**Comment by lo5** [comment]
This can be tackled as part of #18.
